src/dataflow/load.js

```diff
@@ -43,7 +43,7 @@ export function request(target, url, format) {
         df.ingest(target, data, format);
       },
       function(error) {
-        df.error('Loading failed: ' + url, error);
+        df.error('Loading failed: ' + (typeof url === 'string' ? url : JSON.stringify(url)), error);
```

Comment:
Ideally it shouldn't even be a string concatenation -- i think it would be better to have a message key and an object with all params -- `warning('bad-url', { url: url, exception: error })`  (or params could be positional).  This way errors can be easily translatable, and string concatenation does not happen until it is really needed for output (and will be ignored outright if the message is not printed)